changelog/README.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3rd October 2024
+
+* [Inventory Update prices](/channel-manager-operations/inventory.md) extended with `dependent rates`.
```

Comment:
As a general rule, the name of the reference should be the same as the thing that it is referencing, eg. "Update prices" for a reference to the API Operation "Update prices". However, uniquely on the Channel Manager API it's good to add either "CHM:" (Channel Manager side) or "Mews:" (Mews side) to the front, because the API is in two halves, and sometimes operations have the same name on both sides.
```suggestion
* [CHM: Update prices](/channel-manager-operations/inventory.md) extended with `dependent rates`.
```